sync/class.jetpack-sync-options.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/**
+ * Simple class to read/write to the options table, bypassing 
+ * problematic caching with get_option/set_option
+ **/
+
+class Jetpack_Sync_Options {
+
+	static function delete_option( $name ) {
+		global $wpdb;
+		$wpdb->query( $wpdb->prepare( "DELETE FROM $wpdb->options WHERE option_name = %s", $name ) );
+	}
+
+	static function write_option( $name, $value, $autoload = false ) {
```

Comment:
Should we maybe use a different method name here to match the current WordPress methods?